tools/arch-visualizer/viewer/src/store.ts

```
@@ -0,0 +1,244 @@
+import { create } from "zustand";
+import type {
+  Architecture,
+  Component,
+  BreadcrumbItem,
+  ViewMode,
+  Panel,
+  Symbol,
```

Biome (2.3.13)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)